openapi.yml

```diff
@@ -3660,6 +3660,273 @@ paths:
+        - Packages
+      responses:
+        '200':
+          description: OK
+          headers: {}
+          content:
+            application/json:
+              schema:
+                $ref: ./models/PaginatedPackagesList.v1.yaml
+              examples:
+                example:
+                  value:
+                    packages:
+                      - id: 70
+                        package_state: picked_up
+                        recipient:
+                          id: 5
+                          email: tsala@example.com
+                          first_name: Tatiane
+                          last_name: Sala
+                          mobile_number: (604)111-1111
+                          profile_pic_url: 'https://i.pravatar.cc/'
+                          department: Development
+                        location:
+                          name: Vancouver
+                          id: 2
+                        carrier_name: Purolator
+                        created_at: '2020-08-11T09:30Z'
+                        picked_up_at: '2020-08-11T010:30Z'
+                        image:
+                          full_size_url: 'https://i.pravatar.cc/'
+                      - id: 71
+                        package_state: processing
+                        recipient: null
+                        location:
+                          name: Burnaby
+                          id: 3
+                        carrier_name: null
+                        created_at: '2020-08-09T09:30Z'
+                        picked_up_at: null
+                        image:
+                          full_size_url: 'https://i.pravatar.cc/'
+                      - id: 72
+                        package_state: processing
+                        recipient: null
+                        location:
+                          name: Vancouver
+                          id: 2
+                        carrier_name: null
+                        created_at: '2020-08-08T09:30Z'
+                        picked_up_at: null
+                        image:
+                          full_size_url: 'https://i.pravatar.cc/'
+                      - id: 73
+                        package_state: needs_attention
+                        recipient: null
+                        location:
+                          name: Default
+                          id: 1
+                        carrier_name: null
+                        created_at: '2020-08-0709:30Z'
+                        picked_up_at: null
+                        image:
+                          full_size_url: 'https://i.pravatar.cc/'
+                      - id: 74
+                        package_state: recipient_matched
+                        recipient:
+                          id: 4
+                          email: vbaldys@example.com
+                          first_name: Veronica
+                          last_name: Baldys
+                          mobile_number: (604)999-5982
```

Comment:
fake, right?

Reply:
Yeah, I really suggest you change this in case it is your phone number since it will be part of our API documentation. Unless you want to be part of the CS team to answer customer questions.